minifi-toolkit/minifi-toolkit-configuration/src/main/java/org/apache/nifi/minifi/toolkit/configuration/ConfigMain.java

```diff
@@ -336,6 +329,16 @@ public int transform(String[] args) {
         return SUCCESS;
     }
 
+    protected void validateAndPrintIssues(ConfigSchema configSchema) {
+        if (!configSchema.isValid()) {
+            System.out.println("There are validation errors with the template, still outputting YAML but it will need to be edited.");
+            configSchema.getValidationIssues().forEach(System.out::println);
```

Comment:
@JPercivall the first printout looks right to me.  You're missing the name in v1 which gets cleared.  Then when it tries to create the v2 schema, it sees the id missing.

I can do similar logic with the upgrade command as well though.